bin/MT_LoadCheck.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Aug 29 10:37:32 2016
+
+@author: kun
+"""
+
+import robot as rob
+import time
+
+CMD_TIMEOUT = 500000
+
+
+try:
+
+    cmds=['Initialize','Cooldown']
+    for scmd in cmds:
+        tskStat, sampleStat, exception = rob.runATask(scmd, CMD_TIMEOUT)
+        if (tskStat.lower()!="done"):
+            raise Exception(scmd+" "+tskStat+" with exception "+exception)
+
+    cmds=['MT_loadCheck']
+    for i in range(1,2):
+        print( "================================== Plate",i, "=================================")
+
+        for n in range(1,49):
+            rob.par.execute("nSample",n)
+            rob.parChk.execute("nDummy")
+            if n!=int(float(rob.parChk.execute("nSample")))
```

Comment:
Better, but is it still possible that this will round down (ie int takes 1.9999994 ->1 instead of 2)?

Reply:
we are getting back a rounded floating point value from the underlying database as a string - so "2.50" or "2.0". I agree that this handling isn't ideal, but it is probably the best we can do with a general mechanism of returning numbers and imprecise floating point number formats.